app/app.go

```diff
@@ -1038,19 +1038,51 @@ func (app *App) LastLogs(lines int, filterLog Applog) ([]Applog, error) {
 	return logs, nil
 }
 
+type Filter struct {
+	Name      *regexp.Regexp
+	Platform  string
+	TeamOwner string
+	UserOwner string
+}
+
```

Comment:
Hi @lucasweiblen, this solution wouldn't actually work, because that would ask MongoDB to give apps that have platform == "", the solution you proposed is the right one.